examples/snells_law.py

```diff
@@ -16,10 +15,8 @@ class Medium(Enum):
     Air = 1.0003
 
 
-@magicgui(call_button="calculate", result={"disabled": True, "fixedWidth": 100})
-def snells_law(
-    aoi=30.0, n1=Medium.Glass, n2=Medium.Water, degrees=True,
-):
+@magicgui(call_button="calculate", result_widget=True)
```

Comment:
There's something funny in this example where the "result" doesn't seem to update until I shift focus away from the widget to another application, then it shows up. I'm on a mac. Can try and do more debugging / system profiling if needed. Happens with `pythonw` too.

![Uploading magicgui_snell.gif…]()

Reply:
hmmm, I can't repeat this... maybe be something different between our envs/OS?

Reply:
yeah, i'm not so concerned about this, but funny that I don't see it in the widget_demo too ...

Reply:
I can't reproduce it as well (Linux).

Reply:
thanks for checking!